src/renderer/store/editor.js

```diff
@@ -12,6 +12,7 @@ const state = {
   },
   // user preference
   theme: 'light',
+  editorFontFamily: 'Monaco',
```

Comment:
Can you please change default font family to `Open Sans`.

Reply:
Ok